### ISSUE TITLE
Ensure no external compaction metadata propagates when a tablet splits/merges

### ISSUE THREAD

**Issue by keith-turner** [apache/accumulo#2011]

**Comment by milleruntime** [comment]
As a reference for others, here is the point in the code where we cancel compactions. 
It is called while closing the Tablet in `initiateClose(true)` during `Tablet.split()`:
https://github.com/apache/accumulo/blob/9b504121dd69bf033dced56adb55d727b0eafdc4/server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java#L1204

**Comment by keith-turner** [comment]
@dlmarion  I made a few tweaks to you new ITs in b7283aa5853aa62fc30e082d170536c2c810f8c0 that test the changes in c478645f984ca0eee8533073d545a20ced29a77d.  If those look good I think we can close this.   One thing I did was make sure the set of ext compaction ids differ before and after split/merge.  Thinking that new ext compaction ids could possibly show up after the split/merge for the new tablet.
